test/integration/functional_test.go

```diff
@@ -573,6 +578,33 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// Setup extra file for file sync test (validateFileSync), this
+// need to be done prior to starting up minikube.
+func validateCopySynFile(ctx context.Context, t *testing.T, profile string) {
+	// 1. copy random file to MINIKUBE_HOME/files/etc
+	tempfile := filepath.Join(localpath.MiniPath(), "/files/etc/filesync.txt")
```

Comment:
Seeing a .txt file is weird in /etc ... so, perhaps instead of `filesync.txt`, maybe a clearer test-filename like `sync.test`, so that folks seeing a future test error are less likely to think it's a normally expected file?